packages/node-integration-tests/utils/run-tests.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import childProcess from 'child_process';
+import os from 'os';
+
+const testPaths = childProcess.execSync('jest --listTests', { encoding: 'utf8' }).trim().split('\n');
+
+let testsSucceeded = true;
+const testAmount = testPaths.length;
```

Comment:
```suggestion
const numTests = testPaths.length;
```

(Technically, you can only use "amount" with mass nouns (like "milk"), not count nouns (like "cows").)

Reply:
Oh, just saw that you already set it on auto-merge. I can fix in a separate PR.

Reply:
Ah right... Thanks :D d697937f1b6d6b2eb2ec507e245cb7910d48c05f